TotalSegmentator/TotalSegmentator.py

```diff
@@ -98,6 +98,9 @@ def setup(self):
         self.ui.inputVolumeSelector.connect("currentNodeChanged(vtkMRMLNode*)", self.updateParameterNodeFromGUI)
         self.ui.fastCheckBox.connect('toggled(bool)', self.updateParameterNodeFromGUI)
         self.ui.useStandardSegmentNamesCheckBox.connect('toggled(bool)', self.updateParameterNodeFromGUI)
+        self.ui.pullMasterBranchCheckBox.connect('toggled(bool)', self.updateParameterNodeFromGUI)
```

Comment:
I don't think we pull or even use git. It would be more accurate to call this option installLatestDevelopmentChecBox or something similar.